apps/meteor/ee/client/omnichannel/ContactManagerInfo.js

```diff
@@ -24,13 +25,15 @@ function ContactManagerInfo({ username }) {
 
 	return (
 		<>
-			<UserCard.Info className={wordBreak} flexShrink={0} style={{ display: 'flex' }}>
+			<Info className={wordBreak} style={{ display: 'flex' }}>
 				<UserAvatar title={username} username={username} />
-				<UserCard.Username mis='x10' name={username} status={<UserStatus status={status} />} />
-				<Box display='flex' mis='x7' mb='x9' align='center' justifyContent='center'>
-					({name})
-				</Box>
-			</UserCard.Info>
+				<AgentOrContactDetails mis='x10' name={username} status={<UserStatus status={status} />} />
+				{name && (
+					<Box display='flex' mis='x7' mb='x9' align='center' justifyContent='center'>
+						({name})
+					</Box>
+				)}
+			</Info>
```

Comment:
do you agree with me that these component names all seem mismatched?
```
Info
	AgentOrContactDetails
	Box
```
should be something
```
AgentInfo
	AgentInfoDetails
	AgentInfoName

Reply:
Alright makes sense. I've renamed that component. Please check